config/crds/hive_v1alpha1_clusterdeployment.yaml

```diff
@@ -46,6 +46,32 @@ spec:
               description: BaseDomain is the base domain to which the cluster should
                 belong.
               type: string
+            certificateBundles:
+              description: CertificateBundles is a list of certificate bundles associated
+                with this cluster
+              items:
+                properties:
+                  generate:
+                    description: Generate indicates whether this bundle should be
```

Comment:
the way this description currently reads can be interpreted that `generate: true` might mean self-signed certs. maybe update the text to something like:
Generate indicates whether this bundle should have real (ie non-self-signed) certificates generated.

Reply:
Fixed